messages/lightning.preview.app.md

```diff
@@ -85,70 +83,75 @@ Before proceeding, install the self-signed certificate on your device. The certi
 
 # certificate.installation.steps.android.nav-target-api-26-27
 
-Navigate to `Settings > Security & Location > Encryption & credentials` and tap on `Install from SD card` under `Credential storage`.
+Navigate to `Settings > Security & Location > Encryption & credentials` and click `Install from SD card` under `Credential storage`.
 
 # certificate.installation.steps.android.nav-target-api-28
 
-Navigate to `Settings > Security & Location > Advanced > Encryption & credentials` and tap on `Install from SD card` under `Credential storage`.
+Navigate to `Settings > Security & Location > Advanced > Encryption & credentials` and click `Install from SD card` under `Credential storage`.
 
 # certificate.installation.steps.android.nav-target-api-29
 
-Navigate to `Settings > Security > Encryption & credentials` and tap on `Install from SD card` under `Credential storage`.
+Navigate to `Settings > Security > Encryption & credentials` and click `Install from SD card` under `Credential storage`.
 
 # certificate.installation.steps.android.nav-target-api-30-32
 
-Navigate to `Settings > Security > Encryption & credentials` and tap on `Install a certificate` under `Credential storage`. Now tap on `CA certificate` and on `Install anyway`.
+Navigate to `Settings > Security > Encryption & credentials` and click `Install a certificate` under `Credential storage`. Click `CA certificate`, and then click `Install anyway`.
 
 # certificate.installation.steps.android.nav-target-api-33
 
-Navigate to `Settings > Security > More security settings > Encryption & credentials` and tap on `Install a certificate` under `Credential storage`. Now tap on `CA certificate` and on `Install anyway`.
+Navigate to `Settings > Security > More security settings > Encryption & credentials` and click `Install a certificate` under `Credential storage`. Click `CA certificate`, and then click `Install anyway`.
 
 # certificate.installation.steps.android.nav-target-api-34-up
 
-Navigate to `Settings > Security & Privacy > More security & privacy > Encryption & credentials` and tap on `Install a certificate` under `Credential storage`. Now tap on `CA certificate` and on `Install anyway`.
+Navigate to `Settings > Security & Privacy > More security & privacy > Encryption & credentials` and click `Install a certificate` under `Credential storage`. Click `CA certificate`, and then click `Install anyway`.
 
 # certificate.waiting
 
-After installing the certificate, press any key to continue...
+After you install the certificate, press any key to continue...
 
 # mobileapp.notfound
 
-%s is not installed on your device.
+%s isn't installed on your device.
 
 # mobileapp.download
 
-%s is not installed on your device. Do you want to download and install it
+%s isn't installed on your device. Do you want to download and install it?
 
 # examples
 
-- <%= config.bin %> <%= command.id %>
+- Preview the target org "myOrg" in a desktop environment:
```

Comment:
Hmm, not sure what it means to "Preview an org".  I thought this command was for previewing an _app_.  Question: if you don't specify the app name with `--name`, which app does the preview show?  Is there a "default" app?  Or maybe the "first" one (whatever that means)?  Again, my questions show my lack of knowledge!  But if possible, try to rewrite this so you're talking about previewing _apps_ and not orgs.  For example:

Preview the default app in the target org "myOrg" in a desktop environment:

Reply:
- I think "target org" is a holdover from when `preview app` was `preview org`.
- The "default" app is just whatever app is currently configured to show up when you login. Basically, without `--name`, we don't qualify the launch, and just let the browser experience resolve as it will.